ThreeBotPackages/tfgrid_solutions/any_flist/chatflows/your_flist.py

```diff
@@ -0,0 +1,105 @@
+from Jumpscale import j
+import netaddr
+
+
+def chat(bot):
+    """
+    """
+
+    user_info = bot.user_info()
+    name = user_info["username"]
+    email = user_info["email"]
+    ips = ["IPV6", "IPV4"]
+    env = dict()
+
+    explorer = j.clients.threebot.explorer
+    if not email:
+        raise j.exceptions.BadRequest("Email shouldn't be empty")
+
+    form = bot.new_form()
+    flist = form.string_ask(
+        "Please add link of your flist to deploy it for example: https://hub.grid.tf/usr/example.flist"
```

Comment:
let's change this to:

"Please add the link to your flist to be deployed. For example..."